pkg/virt-operator/install-strategy/create.go

```diff
@@ -500,6 +501,56 @@ func createOrUpdateRoleBinding(rb *rbacv1.RoleBinding,
 	return nil
 }
 
+func createOrUpdatePsp(psp *pspv1b1.PodSecurityPolicy,
+	kv *v1.KubeVirt,
+	imageTag string,
+	imageRegistry string,
+	stores util.Stores,
+	clientset kubecli.KubevirtClient,
+	expectations *util.Expectations) error {
+
+	var err error
+	pspcli := clientset.PolicyV1beta1()
+
+	var cachedPsp *pspv1b1.PodSecurityPolicy
+
+	kvkey, err := controller.KeyFunc(kv)
+	if err != nil {
+		return err
+	}
+
+	psp = psp.DeepCopy()
```

Comment:
can you use a different name here
`newPsp := psd.DeepCopy()`

Reply:
this should be fine, he's following the same pattern that's elsewhere in the file. 

I think the logic here is to override the original psp object with a copy to avoid accidentally modifying something from cache unintentionally.